src/main/cpp/src/genomicsdb/query_variants.cc

```diff
@@ -193,10 +196,10 @@ VariantQueryProcessor::VariantQueryProcessor(VariantStorageManager* storage_mana
   clear();
   m_storage_manager = storage_manager;
   m_ad = storage_manager->open_array(array_name, &vid_mapper, "r");
-  if (m_ad < 0)
-    throw VariantQueryProcessorException("Could not open array "+array_name
-                                         +" at workspace: "+storage_manager->get_workspace()
-                                         + "\nTileDB error message : "+tiledb_errmsg);
```

Comment:
Is the motivation for removing the TileDB error message here that it's too implementation specific to be of use to the end user?

Reply:
Yes, but not because it may not be of use to the end use. The error message is already getting logged in the [VariantStorageManager](https://github.com/GenomicsDB/GenomicsDB/blob/7973ab6803f00fb7a13db5af28b67a6c4e40ddd7/src/main/cpp/src/genomicsdb/variant_storage_manager.cc#L547) that is implementation specific(only TileDB!).